kubernetes/Makefile

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Remove this empty line.

Reply:
Done.

Reply:
Heh, I see that github found some other, random line and moved my comment there once you removed the original one. Interesting.